package.json

```diff
@@ -69,8 +69,8 @@
   ],
   "dependencies": {
     "lodash": "3.10.1",
-    "react": "latest",
-    "react-native": "https://github.com/alloy/react-native.git#v0.42.0-with-customisable-source-exts",
+    "react": "16.0.0-alpha.4",
+    "react-native": "file:../react-native",
```

Comment:
I can change this to use the correct fork once I update the fork to v0.43.0-rc.4.